test/react/navbar.test.tsx

```diff
@@ -0,0 +1,52 @@
+import * as React from "react";
+import MyNavbar from "../../app/javascript/components/my_navbar";
+import { expect, test } from "@jest/globals";
+import { getByTestId, queryByText, getByText, render, RenderResult } from "@testing-library/react";
+import "@testing-library/jest-dom/extend-expect";
+import "@testing-library/jest-dom";
+import "jest-extended";
+
+const renderNavbar = (
+  loggedin = true,
+  regPath = "/signup",
+  homePath = "/"
+) : RenderResult => {
+  return render(
+    <MyNavbar logged_in= {loggedin} regPath={regPath} homePath={homePath} />
+  );
+};
+
+test("Component renders as expected", () => {
+  const { container } = renderNavbar();
+  expect(container).toMatchSnapshot();
+});
+
+test("Title button has been passed the right link", () => {
+  const { container } = renderNavbar();
+  expect(getByTestId(container, "HomeButton")).toHaveProperty("href", "http://localhost/");
```

Comment:
This test will fail if your base URL changes, eg. if you pushed it to production. It's not a blocker at this time but can you think of any way to write this test so that it doesn't pose this issue?

Reply:
I've tried looking into this but havent found any answers